Bottom: 927284851a5177273aae3c48a31352d5cc809fd2
Top:    5a04a4c79da561e37953356c3cd2cfc753bc7603
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 16:44:17 -0500

Enable TCP interface of docker


---
diff --git a/roles/docker-common/templates/daemon.json.j2 b/roles/docker-common/templates/daemon.json.j2
index 57de7390..6f6a7376 100644
--- a/roles/docker-common/templates/daemon.json.j2
+++ b/roles/docker-common/templates/daemon.json.j2
@@ -1,4 +1,5 @@
 {
   "log-driver": "json-file",
-  "log-opts": {"max-size": "10m", "max-file": "3"}
+  "log-opts": {"max-size": "10m", "max-file": "3"},
+  "hosts": ["unix:///var/run/docker.sock", "tcp://{{ ansible_default_ipv4["address"] }}:2375"]
 }
